libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1702,6 +1706,27 @@ bool RC_Channel::do_aux_function(const AUX_FUNC ch_option, const AuxSwitchPos ch
     }
 #endif
 
+
+#if AC_PRECLAND_ENABLED
+    case AUX_FUNC::PLND: {
+        AC_PrecLand *_precland = AP::ac_precland();
+        if (_precland == nullptr) {
+            // precland not enabled
+            break;
+        }
+
+        switch (ch_flag) {
+        case AuxSwitchPos::HIGH:
+            _precland->plnd_active(true);
+            break;
+        case AuxSwitchPos::MIDDLE:
```

Comment:
```suggestion
        case AuxSwitchPos::MIDDLE:
            break;
```

this allows us to use the middle position for something else later.